File: test_sl.py
from playwright.sync_api import sync_playwright
with sync_playwright() as p:
    browser = p.chromium.launch(headless=True)
    page = browser.new_page()
    page.goto('https://www.srilankan.com/en_uk/plan-and-book/manage-booking')
    try:
        ln = page.get_attribute('#lastname2refx', 'placeholder')
        pnr = page.get_attribute('#bookref2refx', 'placeholder')
        print("lastname2refx:", ln)
        print("bookref2refx:", pnr)
    except Exception as e:
        print(e)
    browser.close()

File: test_sl.js
const { chromium } = require('playwright');
(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  await page.goto('https://www.srilankan.com/en_uk/plan-and-book/manage-booking');
  const lnPlaceholder = await page.getAttribute('#lastname2refx', 'placeholder').catch(()=>null);
  const pnrPlaceholder = await page.getAttribute('#bookref2refx', 'placeholder').catch(()=>null);
  console.log("lastname2refx:", lnPlaceholder);
  console.log("bookref2refx:", pnrPlaceholder);
  await browser.close();
})();
